gdk4/Gir.toml

```diff
@@ -243,27 +243,19 @@ generate_builder = false
 name = "Gdk.ContentDeserializer"
 status = "generate"
     [[object.function]]
-    name = "get_input_stream"
-        [object.function.return]
-        nullable = false
-    [[object.function]]
-    name = "get_mime_type"
-        [object.function.return]
-        nullable = false
-    [[object.function]]
     name = "get_gtype"
     rename = "type_"
     [[object.function]]
+    name = "get_cancellable"
+        [object.function.return]
+        nullable = true # TODO: fixme upstream
```

Comment:
Please create an issue before merging

Reply:
Also for the others

Reply:
Sent a PR https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/4310

Reply:
Can you put a link to that into `Gir.toml`?

Reply:
I don't plan on merging it yet, still working on the gtk ones